assets/js/wcmp-admin.js

```diff
@@ -784,22 +783,32 @@ jQuery(function($) {
     }
 
     request.afterDone = function(response) {
-      openPdf(response);
-    };
+      if (response.includes('PDF') && wcmp.download_display === 'display') {
+        handlePDF(request);
+      }
 
-    if (wcmp.download_display === 'download') {
-      doRequest.bind(button)(request);
-    } else {
-      var url;
+      if (response.includes('api.myparcel.nl') && wcmp.download_display === 'download') {
```

Comment:
Bij voorkeur dit soort checks in variabelen stoppen met een goed leesbare naam.

Reply:
done